x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -261,6 +262,27 @@ export const RulesList = ({
     setCurrentRuleToEdit(ruleItem);
   };
 
+  const onRunRule = async (id: string) => {
```

Comment:
nit: maybe we could extract this out into a function and reuse it between here and the details view? Just so if we change the behavior in one location, the two actions stay in sync

Reply:
Resolved in https://github.com/elastic/kibana/pull/139848/commits/8d739ac9c783f035e6dba8bd3dfe1b20425b173f